src/main/java/org/broadinstitute/hellbender/utils/help/GATKHelpDocWorkUnitHandler.java

```diff
@@ -58,4 +64,27 @@ protected void addCustomBindings(final DocWorkUnit currentWorkUnit) {
         }
     }
 
+//    @Override
```

Comment:
Can this be removed since it's commented out?

Reply:
Yeah, done.